src/metabase/search/appdb/index.clj

```diff
@@ -154,8 +175,10 @@
       (specialization/batch-upsert! table-name entries)
       (catch Exception e
         ;; ignore database errors, the table likely doesn't exist, or has a stale schema.
-        (when-not (instance? PSQLException (ex-cause e))
-          (throw e))))))
+        (let [cause (ex-cause e)]
+          (when-not (or (instance? PSQLException cause)
+                        (instance? ServerErrorMessage cause))
```

Comment:
omg I love how detailed documentation is about `ServerErrorMessage`. I see it's from postgres, but when does it occur?

Reply:
I think it's actually the "cause" of the `PSQLException`, I just got confused still seeing it in a stacktrack in CI. I think it was being thrown by the *next* database operation, as this exception cancels the transaction.

Locally both bad schema and missing table result in `PSQLException`.

Reply:
Maybe it makes sense to remove it then? I mean not like it's necessary to remove, but it'll be a bit less surprising to see.